src/_includes/components/apply-note.njk

```diff
@@ -0,0 +1,21 @@
+<div style="display: flex;">
+  <div>Octogons shapes go here</div>
+  <div>
+    <p>When you’re ready to apply we need two things:</p>
+    <ol>
+      <li>A completed questionare</li>
+      <p>
+        Follow the instructions this <a href="">Word Doc</a> or
+        <a href="">Google Doc</a> or (both are the same, use the format that
+        works best for you)
+      </p>
+      <li>A completed application</li>
+      <p>
+       Available at 
+        <a href="ideas42.applytojob.com/apply/KsR4Qk11of"/>
```

Comment:
This `a` has an extra `/` on the opening tag. It's causing the link not to show up on the page

Reply:
Fixed!